filetests/isa/x86/simd-avx512-arithmetic-binemit.clif

```diff
@@ -0,0 +1,17 @@
+test binemit
+set enable_simd
+target x86_64 skylake has_avx512dq=true
+
+function %imul_i64x2() {
+block0:
+    [-, %xmm1]    v0 = vconst.i64x2 [1 2]
+    [-, %xmm2]    v1 = vconst.i64x2 [2 2]
+    [-, %xmm19]   v2 = imul v0, v1 ; bin: 62 e2 f5 08 40 da
+    ; 62, mandatory EVEX prefix
+    ; e2 = 1110 0010, R, X, B are unset (inverted) while R' is set (MSB in %xmm19); mm is set to 0F38
```

Comment:
I am a little :crying_cat_face: that the manual (at least the latest one I see online: https://software.intel.com/en-us/download/intel-64-and-ia-32-architectures-sdm-combined-volumes-1-2a-2b-2c-2d-3a-3b-3c-3d-and-4 from October 2019) doesn't seem to say anything about these bits being inverted in EVEX prefixes. In fact, just reading the manual suggests the opposite: `Combine with ModR/M.reg, ModR/M.rm (base, index/vidx)`, from `Table 2-30.  EVEX Prefix Bit Field Functional Grouping`.

Anyway, I see the negated interpretation is [backed up by XED](https://github.com/intelxed/xed/blob/master/src/dec/xed-ild.c#L1281-L1287) so :+1:

Reply:
Yeah, me too... mostly because I implemented it without inversion first and then had to invert it after getting weird results. I will ask around here to see if anyone knows what team is responsible for editing the manual.

Reply:
Ok, I e-mailed someone internally about this. Let's see what happens.